tensorflow_model_optimization/g3doc/guide/combine/cqat_example.ipynb

```diff
@@ -597,7 +598,7 @@
       "source": [
         "## Apply post-training quantization and compare to CQAT model\n",
         "\n",
-        "Next, we use post-training quantization (no fine-tuning) on the clustered model and check its accuracy against the CQAT model. This demonstrates why you would need to use CQAT to improve the quantized model's accuracy.\n",
+        "Next, we use post-training quantization (no fine-tuning) on the clustered model and check its accuracy against the CQAT model. This demonstrates why you would need to use CQAT to improve the quantized model's accuracy. The difference could be not very visible, because the MNIST model is quite small and overparametrized.\n",
```

Comment:
-> "The difference may not be very visible, because the MNIST model is quite small and overparametrized.